app/services/isolate.rb

```diff
@@ -243,8 +245,7 @@ def directory_bindings options = {}
       binding = opt.unshift(fullpath).join(':')
       next "--dir=#{boxpath}=" unless File.exist?(fullpath)
       "--dir=#{boxpath}=#{binding}"
-    end.compact +
-      ["--dir=/tmp=#{expand_path('tmp')}:rw:noexec"] # link /tmp to /box/tmp (ghc expects a temporary directory)
```

Comment:
I removed this as it seems to cause issues, since the `/tmp` directory is now created by default (https://github.com/ioi/isolate/commit/715add21d25e066df6420d80917b954432f74d6b)